data/tribes/initialization/frisians/units.lua

```diff
@@ -1932,7 +1936,7 @@ wl.Descriptions():new_tribe {
             -- TRANSLATORS: Purpose helptext for a Frisian production site: Tavern
             purpose = pgettext("building", "Prepares rations to feed the scouts and miners."),
             -- TRANSLATORS: Performance helptext for a Frisian production site: Tavern
-            performance = pgettext("frisians_building", "The tavern can produce one ration in %s on average if the supply is steady; otherwise, it will take 50%% longer."):bformat(format_seconds(33))
+            performance = pgettext("frisians_building", "The tavern can produce one ration in %s on average if the supply is steady; otherwise, it will take 50%% longer."):bformat(format_seconds(37))
```

Comment:
"Fully supplied" instead of steady, should state time instead of "50% longer", and explain the condition better, something like:

"If the tavern is fully supplied, it can produce on ration in %1s on average, using 2 kinds of ingredients. If it can only use one kind of ingredient, it can produce one ration in %2s on average."

(the explanation can be seen in the production section)